app/routes/planet_routes.py

```diff
@@ -0,0 +1,49 @@
+from flask import Blueprint, jsonify, abort, make_response
+
+class Planet:
+    def __init__(self, id, name, description, has_moon=None):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.has_moon = has_moon
+
+    def make_dict(self):
+        return dict(
+                id = self.id,
+                name = self.name,
+                description = self.description,
+                has_moon = self.has_moon,  
+            )
+
+planets = [
+    Planet(1, "Mercury", "terrestrial", False),
+    Planet(2, "Jupiter", "gaseous", True),
+    Planet(3, "Earth", "terrestrial", True)
+]
+
+bp = Blueprint("planets_bp",__name__, url_prefix="/planets")
+
+# GET /planets
+@bp.route("", methods=["GET"])
+def list_planets():
```

Comment:
👍 

Get all looks good. I like the function name you used too. Nice use of the list comprehension and instance method to build the dictionary for the instance.